operator/src/test/java/org/bf2/operator/operands/KafkaClusterTest.java

```diff
@@ -216,6 +216,18 @@ private void alternativeConfig(Consumer<KafkaInstanceConfiguration> configModifi
         QuarkusMock.installMockForType(mock, KafkaInstanceConfigurations.class);
     }
 
+    @Test
+    void testElevatedPriority() throws IOException {
+        final ManagedKafka managedKafka = exampleManagedKafka("2Ti");
+
+        Mockito.when(overrideManager.useElevatedPriority(Mockito.anyString())).thenReturn(true);
+
+        Kafka kafka = kafkaCluster.kafkaFrom(managedKafka, null);
+
+        assertEquals("kas-fleetshard-core", kafka.getSpec().getKafka().getTemplate().getPod().getPriorityClassName());
```

Comment:
I guess these need to be updated to new name to fix the unit tests

Reply:
Yes, sorry things were incomplete.  Should be good now.